cmd/knit/README.md

```diff
@@ -0,0 +1,140 @@
+knit usage examples
+===================
+
+All examples captured on a 4 cores (with HT, thus 2 physical cores) laptop.
+
+
+checking CPU affinities of all processes. Here we grep `knit` for the sake of the brevity, the output
+would be very verbose.
+```bash
+knit cpuaff | grep knit
+PID 166574 (knit                            ) TID 166574 (knit            ) can run on [0 1 2 3]
+PID 166574 (knit                            ) TID 166576 (knit            ) can run on [0 1 2 3]
+PID 166574 (knit                            ) TID 166577 (knit            ) can run on [0 1 2 3]
+PID 166574 (knit                            ) TID 166578 (knit            ) can run on [0 1 2 3]
+PID 166574 (knit                            ) TID 166579 (knit            ) can run on [0 1 2 3]
+PID 166574 (knit                            ) TID 166580 (knit            ) can run on [0 1 2 3]
+```
+
+Checking which processes can run on CPU #3. There is no affinity constraint, all process can run there.
+Again we grep `lnit` for the sake of brevity.
+```bash
+$ ./_output/knit cpuaff -C 3 | grep knit
+PID 166617 (knit                            ) TID 166617 (knit            ) can run on [3]
```

Comment:
Why not print all the cores here as well? Maybe with the filtered cores first?

Reply:
Well maybe tht is not such a great idea when you have 100 cores.. in any way, can this collapse consecutive cores?

Reply:
not sure what's the ask here. The idea is you always give `-C`, which is the isolated core. We can totally format the allowed cpu list in cpulist (7) format